__tests__/buildTranslationFiles.spec.ts

```diff
@@ -99,17 +102,17 @@ function removeI18nFolder(type: TranslationCategory) {
   fs.removeSync(`./${sourceRoot}/${type}/i18n`);
 }
 
-const formats: FileFormats[] = ['pot', 'json'];
+// const formats: FileFormats[] = ['pot', 'json'];
```

Comment:
why?

Reply:
It's a mistake and an oversight on my part during my testing/debugging phase, sorry. I revert.